### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by dongqian0206** [analvikingur/pytorch_RVAE#11]
Hi, 

Would you like to tell me why you added scalar '79' in front of the cross entropy (RVAE, Line 110)?
loss = 79 * cross_entropy + kld_coef(i) * kld

best

**Comment by nguyenvo09** [comment]
when I remove 79, kld can go to zero very fast.

**Comment by dongqian0206** [comment]
Hi, thanks for your reply. We have to use kl annealing to alleviate kl vanishing issue in this model. I think it is the same.

best
